### PR TITLE
NCL 799 - Define a Product Version - Web UI

### PR DESCRIPTION
**Pull request by vibe13**

### CODE REVIEW

**Comment by vibe13** [comment]
@rnc @alexcreasy please review

File: pnc-ui/app/product/views/product.version.html [hunk 1]
```diff
@@ -2,8 +2,8 @@
   <pnc-header>
     <pnc-header-title>{{ versionCtrl.version.version || 'Empty' }}</pnc-header-title>
     <pnc-header-buttons>
-      <button type="button" class="btn btn-lg btn-default disabled" data-toggle="tooltip" title="Edit Version">
-        <i class="pficon pficon-edit"></i> Edit
+
+      <button type="button" class="btn btn-lg btn-default" data-toggle="tooltip" title="Edit Produc Versiont" ng-click="productVersionForm.$show()" ng-class="{ 'active': productVersionForm.$visible }"><i class="pficon pficon-edit"></i> Edit
```
Comment (rnc) [inline pnc-ui/app/product/views/product.version.html]: Should the title be Edit Product Version ?

Comment (vibe13) [inline pnc-ui/app/product/views/product.version.html]: yes thanks, fixed

**Comment by vibe13** [comment]
retest this please

**Comment by vibe13** [comment]
retest this please

**Comment by vibe13** [comment]
closing as opened https://github.com/project-ncl/pnc/pull/335 with new rebase

**Pull request closed without merging**